backend-api/app/models/consumers.py

```diff
@@ -0,0 +1,12 @@
+from ..database import Base
+import sqlalchemy as sa
+from . import users
+from sqlalchemy.orm import relationship
+
+class Consumer(users.User, Base):
+    __tablename__ = "Consumers"
+    id = sa.Column(sa.Integer, sa.ForeignKey("Users.id"), primary_key=True, index=True, nullable=False)
```

Comment:
Let's not make the id a foreign key.  We can just add a user field that has the foreign key